docs/modules/ROOT/pages/usage.adoc

```diff
@@ -0,0 +1,103 @@
+= Usage
+
+Druid requires a Zookeeper to run, as well as a database.
+
+
+== Setup Prerequisites
+
+=== Zookeeper operator
+
+Please refer to the https://github.com/stackabletech/zookeeper-operator[Zookeeper] operator and https://docs.stackable.tech/zookeeper/index.html[docs].
+
+=== SQL Database
+
+Druid requires a MySQL or Postgres database.
+
+For testing purposes, you can spin up a PostgreSQL database with the bitnami PostgreSQL helm chart.  Add the bitname repository:
+
+    helm repo add bitnami https://charts.bitnami.com/bitnami
+
+And setup the Postgres database:
+
+    helm install druid bitnami/postgresql \
+    --set postgresqlUsername=druid \
+    --set postgresqlPassword=druid \
+    --set postgresqlDatabase=druid
+
+=== Creating a Druid Cluster
+
+With the prerequisites fulfilled, the CRD for this operator must be created:
+
+    kubectl apply -f /etc/stackable/druid-operator/crd/druidcluster.crd.yaml
+
+Then a cluster can be deployed. Fill in the correct database connection and path for deep storage.
+
+    cat <<EOF | kubectl apply -f -
+apiVersion: druid.stackable.tech/v1alpha1
+kind: DruidCluster
+metadata:
+  name: simple
+spec:
+  version: 0.22.0
+  zookeeperReference:
+    namespace: default
+    name: simple
+  metadataStorageDatabase:
+    dbType: postgresql
+    connString: jdbc:postgresql://druid-postgresql/druid
+    host: druid-postgresql
+    port: 5432
+    user: druid
+    password: druid
+  deepStorage:
+    storageType: local
+    storageDirectory: /path/to/druidDeepStorage
+  brokers:
+    roleGroups:
+      default:
+        selector:
+          matchLabels:
+            kubernetes.io/os: linux
+        config:
+          plaintextPort: 8082
+        replicas: 1
+  coordinators:
+    roleGroups:
+      default:
+        selector:
+          matchLabels:
+            kubernetes.io/os: linux
+        config:
+          plaintextPort: 8081
+        replicas: 1
+  historicals:
+    roleGroups:
+      default:
+        selector:
+          matchLabels:
+            kubernetes.io/os: linux
+        config:
+          plaintextPort: 8083
+        replicas: 1
+  middleManagers:
+    roleGroups:
+      default:
+        selector:
+          matchLabels:
+            kubernetes.io/os: linux
+        config:
+          plaintextPort: 8091
+        replicas: 1
+  routers:
+    roleGroups:
+      default:
+        selector:
+          matchLabels:
+            kubernetes.io/os: linux
+        config:
+          plaintextPort: 8888
+        replicas: 1
+
+The Router is hosting the web UI, which should now be reachable on the Router host on port 8888.
```

Comment:
The port is not accessible by default in a containerized environment. Either we create a service or use something like:

```
 kubectl port-forward --namespace default po/druid-simple-router-default-druidworker3-pk2n4 8888:8888
```
where `druid-simple-router-default-druidworker3-pk2n4` is the pod rounning the router.